README.md

```diff
@@ -1,12 +1,5 @@
 # Rare: The Publishing Platform for the Discerning Writer
 
-## Getting Started
-1. Install dependencies: `npm install`
-2. Run the code `npm start`
-3. Current server can be downloaded [HERE](https://github.com/nss-day-cohort-60/django-taters-api).
-4. This template is using [Bulma](https://bulma.io/documentation) for styling. Take a little bit of time to familiarize yourself with the framework if you would like to continue using it.
-
-
 ## Project Overview:
 Rare Publishing is a Full-Stack group project with collaboration from students in Cohort 60 of Nashville Software School.
 This app was initially supported by a server built with [Python](https://github.com/nss-day-cohort-60/rare-python-server-tyrannical-tators). The backend was then re-built using Django REST framework.
```

Comment:
Can we link to the Django server repository on the Django REST framework line? https://github.com/nss-day-cohort-60/django-taters-api

Reply:
I added the link and also added those features you mentioned.